spec/system/publishers/publishers_can_view_candidate_messages_spec.rb

```diff
@@ -160,4 +160,62 @@
       end
     end
   end
+
+  describe "message ordering" do
```

Comment:
Given the cost of system tests. I wonder if we could assert the conversation order within the `viewing candidate messages` test above and save a specific setup and run dedicated to only test the ordering.